pkgs/top-level/metrics.nix

```diff
@@ -35,7 +35,7 @@ runCommand "nixpkgs-metrics"
       [[ -n $x ]] || exit 1
       echo "$name.allocations $x B" >> $out/nix-support/hydra-metrics
 
-      x=$(sed -e 's/.*values allocated: \([0-9]\+\) .*/\1/ ; t ; d' stats)
+      x=$(sed -e 's/.*values allocated bytes: \([0-9]\+\).*/\1/ ; t ; d' stats)
```

Comment:
I think this was supposed to be `values allocated count`, not bytes: https://hydra.nixos.org/job/nixpkgs/trunk/metrics#tabs-charts

Reply:
Eep, yes.  Sorry about that.

Review was for approval of "requiring" Nix 2 for this (thanks!), will fix shortly.